app/src/ccdashboard/domain/core.cljc

```diff
@@ -93,6 +94,14 @@
                    (call-api {:consultant new-consultant})
                    (mixpanel/track "consultant/search"))))))
 
+
+(defn call-team-stats-api []
+  (ajax/GET "/team-stats"
+      {:handler         handle-team-stats-api-response
```

Comment:
you could pull the common parts of this map and the one of `call-api` out into a `def`ed default map and then just `assoc` the differing parts onto it.